manifests/v/valinet/ExplorerPatcher/Prerelease/22621.3880.66.5/valinet.ExplorerPatcher.Prerelease.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: valinet.ExplorerPatcher.Prerelease
+PackageVersion: 22621.3880.66.5
+InstallerType: inno
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/valinet/ExplorerPatcher/releases/download/22621.3880.66.5_5094108/ep_setup.exe
+  InstallerSha256: 8B9554E2A513AFBBA280C525347034C39F4DB8197BED48CA340E1A7596C7D770

```

> 9/3/2024 7:28:54 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   1C4E1847C722DB18D58216C43AA40AD87C8A38AA6196E69D55C0687B8506BF94
Expected hash: 8b9554e2a513afbba280c525347034c39f4db8197bed48ca340e1a7596c7d770

```suggestion
  InstallerSha256: 1C4E1847C722DB18D58216C43AA40AD87C8A38AA6196E69D55C0687B8506BF94
```